crates/compiler/can/tests/snapshots/test_suffixed__suffixed_tests__pizza_dbg.snap

```diff
@@ -1,14 +1,14 @@
 ---
 source: crates/compiler/can/tests/test_suffixed.rs
-assertion_line: 459
+assertion_line: 473
```

Comment:
This file was accidentally committed early as part of a previous PR. I've updated the source code for the test, so the contents of the snapshot have updated, even though it looks like a new test case.